app/uk/gov/hmrc/merchandiseinbaggage/views/DeclarationConfirmationView.scala.html

```diff
@@ -81,6 +82,16 @@ <h2 id="goodsDetailsId" class="govuk-heading-m" id="main-message">@messages("che
      <dt id="@{s"quantityLabel_$index"}" class="govuk-summary-list__key">@messages("checkYourAnswers.detailsOfTheGoods.quantity")</dt>
      <dd id="@{s"quantity_$index"}" class="govuk-summary-list__value" style="text-transform: capitalize">@goods.categoryQuantityOfGoods.quantity</dd>
     </div>
+
+    @goods match {
+     case good:ImportGoods => {
```

Comment:
any reason why we are not showing  any row in-case of exports ?

Reply:
The base class `Goods` is missing  `producedInEu` so we cannot use that.